charter-drafts/editing-2021.html

```diff
@@ -311,6 +311,8 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             <dd>Overlapping specifications such as Highlight API.</dd>
             <dt><a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures (APA) Working Group</a></dt>
             <dd>Explore accessibility use cases and gap analysis and ensure new specifications provide needed accessibility features.</dd>
+            <dt><a href="https://www.w3.org/International/core/">Internationalization Working Group</a></dt>
+            <dd>There could be many considerations on internationalization point of view to be addressed in editing especially for edit/input, keyboard, etc.</dd>
```

Comment:
There **are** many internationalisation considerations regarding input, changing writing directions an language-specific punctuation and layout conventions, use of different keyboards and virtual keyboards, etc.

Reply:
Thanks @chaals ! Sounds much better :)